openapi.yml

```
@@ -194,10 +195,18 @@ components:
             pruefidentifikator:
               type: string
               example: '25007'
+            versionsnummer:
+              type: string
+              example: '2.1c'
+            veroeffentlichungsdatum:
+              type: string
+              example: '2025-04-17'
           required:
             - description
             - direction
             - pruefidentifikator
+            - versionsnummer
+            - veroeffentlichungsdatum
```

englischnamen sind nicht besser? aber wenn nur nicht zu aufwändig wäre 
```suggestion
            - pid
            - version_number
            - publish_date
```

---

ich wollte mich lieber an die Formulierung aus den XML-Quelldaten halten, damit man das später leichter zuordnen kann.

Sonst hast du natürlich Recht.